### PR TITLE
Feature/atomic tool glue

### PR DESCRIPTION
**Pull request by goern**
added some glue so that we can use this image with the atomic tool

### CODE REVIEW

**Comment by mfojtik** [comment]
LGTM @jhadvig?

File: README.md [hunk 1]
```diff
@@ -79,13 +79,32 @@ You can also set following mount points by passing `-v /host:/container` flag to
...
+  for a set of environment variables:
+
+|    Variable name          |    Value                                    |
+| :------------------------ | -----------------------------------------   |
+|  `MONGODB_USERNAME`       | mongodb-user                                | 
+|  `MONGODB_PASSWORD`       | pa55word                                    |
+|  `MONGODB_DATABASE`       | admpa55word                                 |
+|  `MONGODB_ADMIN_PASSWORD` | testing                                     |
+
+  Additionally data is stored to /var/lib/mongodb-24-centos7/
```
Comment (jhadvig) [inline README.md]: Worth to mention where this directory takes place. On the host or in the container.

File: 2.4/Dockerfile.rhel7 [hunk 2]
```diff
@@ -16,6 +16,13 @@ ENV MONGODB_VERSION         2.4
 ENV IMAGE_DESCRIPTION       MongoDB 2.4
 ENV IMAGE_TAGS              mongodb,mongodb24
 ENV IMAGE_EXPOSE_SERVICES   27017:mongodb
+ENV container docker
+
+LABEL RUN docker run -d -e MONGODB_USERNAME=mongodb-user -e MONGODB_PASSWORD=pa55word -e MONGODB_ADMIN_PASSWORD=admpa55word -e MONGODB_DATABASE=testing -v ${DATADIR}:/var/lib/mongodb/data IMAGE
```
Comment (jhadvig) [inline 2.4/Dockerfile.rhel7:21]: Wont the `LABEL` instruction break our builds ? Since they are not running on Docker 1.6.0

Comment (goern) [inline 2.4/Dockerfile.rhel7:21]: no, they should be ignored by other docker versions and LABELs are supported by docker 1.5-rhel7, therefore should also be supported by the centos build.

File: README.md (hunk 1 above)
Comment (goern) [inline README.md]: ja, will add that line.

File: 2.4/Dockerfile.rhel7 (hunk 2 above)
Comment (jhadvig) [inline 2.4/Dockerfile.rhel7:21]: OK. If its so, then LGTM and thanks ! :)

**Pull request closed without merging**